TrkHitReco/src/CombineStrawHits_module.cc

```diff
@@ -228,6 +231,15 @@ namespace mu2e {
     combohit._tres       = _terr/sqrt(combohit._nsh); // error proportional to # of straws (roughly)
     //      float wvar           = sqrtf((wacc2/weights-wacc/weights*wacc/weights));//define quality as variance/average ratio
     combohit._qual       = 1.0; // quality isn't used and should be removed FIXME
+//-----------------------------------------------------------------------------
+// if combohit._wres is not consistent with the spread of the individual hits,
+// increase it
+//-----------------------------------------------------------------------------
+    if (_checkWres) {
+      float wdist2 = wacc2/weights;
+      float sigw   = sqrt((wdist2-combohit._wdist*combohit._wdist)/(combohit.nCombo()-1+1e-12));
```

Comment:
Since this test only makes sense with nCombo()>1, please just add that as a criteria in the 'if'.  Then you won't need the fudge factor, and you won't get bizarre, huge results.

Reply:
I'm still confused: it seems this cut can be applied even if nCombo=1, in which case the value of sig2 will be huge (~10^12).  Shouldn't the logic be to exclude this case: ie (if (_checkWres && combohit.nCombo() > 1){...?

Reply:
if nCombo=1, the numerator is zero.